mis_builder/models/aep.py

```diff
@@ -328,7 +328,7 @@ def get_aml_domain_for_dates(self, date_from, date_to,
         return expression.normalize_domain(domain)
 
     def do_queries(self, date_from, date_to, period_from, period_to,
-                   target_move):
+                   target_move, additional_move_line_filter=[]):
```

Comment:
Sure, I know the risk. But in this case I never append anything in the method so it is completely safe. Anyway, I'll change.